javascript/webpacker_react-npm-module/dist/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
To be able to use `import * from "webpacker-react/something" we need `something.js` to be in the root of the NPM package. With Babel transpiling, output is created in `dist/`. I havent found an alternative other than running `yarn publish` from the `dist/` directory, and the `package.json` needs to be in this directory (but will only be used for the NPM package, so I stripped it down to the minimum).

I am wondering if `javascript/webpacker_react-npm-module/package.json` (used to install Babel for transpiling, and nothing else) should be moved to the root directory. I am also considering moving the NPM scripts to some Rake tasks to ease building the various parts.